nixos/tests/wireguard/generated.nix

```diff
@@ -0,0 +1,57 @@
+import ../make-test.nix ({ pkgs, ...} : {
+  name = "wireguard-generated";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ ma27 ];
```

Comment:
Would you mind adding yourself as maintainer here as well? :sweat_smile:

Reply:
Done!